backend/modules/dataloaders/truefoundryloader.py

```diff
@@ -82,6 +82,8 @@ def load_filtered_data(
                     if f.startswith("."):
                         continue
                     full_path = os.path.join(root, f)
+                    if ".truefoundry" in full_path:
+                        continue
```

Comment:
is there a case where we would want to iterate over dest_dir instead of just `download_info`

(`download_info` should have been called `artifact_files_dir`)

Then you would not need this check

Reply:
Can call `download_info` as `artifact_files_dir` but then there is `data_dir` as well, what might be a suitable name.
You are write, can walk over download info rather than dest_dir. Will make that change.